webpack.config.js

```diff
@@ -38,6 +39,10 @@ module.exports = {
 	resolve: {
 		symlinks: false, // allow module resolution with `npm link`
 		extensions: ['.js', '.jsx'], // allow leaving off file extension when importing
+		alias: {
```

Comment:
Unfortunately, `webpack@^4` does not support the `exports` field in package.json, so I had to add aliases.